### PR TITLE
Spinner Collisions

### LINKED ISSUE #36: Implement spinner hit collider
Spinners have their own hit objects (actually, they have a few, two circles for the bracket and a special one for the plate).

### PR DESCRIPTION
**Pull request by freezy**
This implements spinner collisions and movement. Closes #36.

Pretty much the same as gates, with some minor changes in the collision and movement code.

### CODE REVIEW

File: VisualPinball.Engine/VisualPinball.Engine.csproj [hunk 1]
```diff
@@ -88,9 +88,18 @@
 		<Reference Include="System.ServiceModel" />
 		<Reference Include="System.Transactions" />
 		<Reference Include="System.Xml" />
+		<Reference Include="Unity.Entities.Hybrid, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
```
Comment (Vroonsh) [inline VisualPinball.Engine/VisualPinball.Engine.csproj:91]: Are these assembly reference needed ? some are pointing to vpx_import_test

File: VisualPinball.Unity/VisualPinball.Unity/VPT/Spinner/SpinnerDisplacementSystem.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+					// 		m_pspinner->FireGroupEvent(DISPID_SpinnerEvents_Spin);
+					// 	}
+					//
+					// } else {
+					// 	if (movementData.AngleSpeed < target) {
+					// 		m_pspinner->FireGroupEvent(DISPID_SpinnerEvents_Spin);
+					// 	}
+					// }
+
+					while (movementData.Angle > 2.0f * math.PI) {
```
Comment (Vroonsh) [inline VisualPinball.Unity/VisualPinball.Unity/VPT/Spinner/SpinnerDisplacementSystem.cs:80]: maybe movementData.Angle = Mathf.Repeat(movementData.Angle, 0.f, 2.f * math.PI) could work also.

File: VisualPinball.Engine/VisualPinball.Engine.csproj (hunk 1 above)
Comment (freezy) [reply VisualPinball.Engine/VisualPinball.Engine.csproj:91]: You're right this has nothing to do here.

File: VisualPinball.Unity/VisualPinball.Unity/VPT/Spinner/SpinnerDisplacementSystem.cs (hunk 2 above)
Comment (freezy) [reply VisualPinball.Unity/VisualPinball.Unity/VPT/Spinner/SpinnerDisplacementSystem.cs:80]: Hmm I don't see the signature of your example in [`Mathf.Repeat()`](https://docs.unity3d.com/ScriptReference/Mathf.Repeat.html).

But we're in Burst land here anyway, so we can't use `Mathf`, and Unity's Burst-compatible `math` lib doesn't seem to have something similar.

Comment (Vroonsh) [reply VisualPinball.Unity/VisualPinball.Unity/VPT/Spinner/SpinnerDisplacementSystem.cs:80]: Yup sorry was Mathf.Repeat(movementData.Angle, 2.f * math.PI), anyway didn't thought about Burst and math lib limitations

**Pull request merged**